src/components/App.js

```diff
@@ -12,27 +15,45 @@ export const styles = {
   },
 };
 
-function App({ classes }) {
-  return (
-    <React.Fragment>
-      <TopBar />
-      <Grid container>
-        <Grid item md={9} sm={12} xs={12} className={classes.calendar}>
-          <CalendarViewContainer />
-        </Grid>
+export const mapDispatchToProps = dispatch => ({
+  fetchSchools: () => {
```

Comment:
You can use [shorthand notation](https://egghead.io/lessons/javascript-redux-using-mapdispatchtoprops-shorthand-notation) for this.

Reply:
```javascript
const mapDispatchToProps = {
  fetchSchools,
};
```